examples/kafka/envoy.yaml

```diff
@@ -0,0 +1,37 @@
+static_resources:
+  listeners:
+  - address:
+      socket_address:
+        address: 0.0.0.0 # Host that Kafka clients should connect to.
+        port_value: 19092  # Port that Kafka clients should connect to.
+    filter_chains:
+    - filters:
+      - name: envoy.filters.network.kafka_broker
```

Comment:
Please add the new config to https://github.com/envoyproxy/envoy/blob/master/examples/BUILD, this will make sure these configs get config tested, which will cause this config to fail on the deprecated config field. It will need to be updated to use typed_config (see other examples).

Reply:
Will do! Thanks for the review.